cfpb/cfgov-refresh
# Update to latest dependencies

## Changes

- Update test menu selectors to work with new menu.
- Removed hover-related menu tests that are no longer applicable.
- Skip pagination tests to make "cucumber" and "cucumber-new" tests the same.
- Removed `gulp test:acceptance` from `gulp test` command.

- Update `@babel/core` from `7.10.2` to `7.10.5`.
- Update `@babel/preset-env` from `7.10.2` to `7.10.4`.
- Update `autoprefixer` from `9.8.0` to `9.8.5`.
- Update `glob-all` from `3.1.0` to `3.2.1`.
- Update `gulp-clean-css` from `4.2.0` to `4.3.0`.
- Update `gulp-modernizr` from `3.4.0` to `4.0.1`.
- Update `less` from `3.10.3` to `3.12.0`.
- Update `svg-inline-loader` from `0.8.0` to `0.8.2`.
- Update `terser-webpack-plugin` from `3.0.1` to `3.0.6`.
- Update `babel-jest` from `26.0.1` to `26.1.0`.
- Update `jest` from `26.0.1` to `26.1.0`.
- Update `jest-cli` from `26.0.1` to `26.1.0`.
- Update `jsdoc` from `3.6.3` to `3.6.4`.
- Update `lighthouse` from `5.6.0` to `6.1.1`.
- Update `minimist` from `1.2.0` to `1.2.5`.
- Update `protractor` from `5.4.2` to `7.0.0`.
- Update `protractor-cucumber-framework` from `6.2.0` to `6.2.1`.
- Update `snyk` from `1.279.1` to `1.362.0`.
- Update `through2` from `3.0.1` to `4.0.2`.

## How to test this PR

1. `gulp docs` should run.
2. `gulp audit:perf` should run.
3. `gulp test:acceptance-new` should pass.
4. `./frontend.sh` should pass and site should function as before.
